projects/examples/src/components/quick-search/quick-search-sync-async.example.component.html

```diff
@@ -16,6 +16,21 @@
         description="The text to be shown in the quick search input field when it is empty"
     >
     </vcd-form-input>
+    <vcd-form-checkbox
```

Comment:
Ideally, we'd have an example for this feature, not have this embedded in an example that is meant to show async results

Reply:
reverted